ccd-definition/AuthorisationComplexType/solicitor.json

```diff
@@ -1 +1,82 @@
-[]
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseFieldID": "applicantSolicitor1OrganisationPolicy",
```

Comment:
Do we need changes to this file? Permissions for complex type fields should be inherited from parent case field. Or is there any subfield we're omitting here so it gets hidden?

Reply:
It's given in example definition, so following it